models/Post.php

```diff
@@ -340,13 +342,14 @@ public function getSummaryAttribute()
             return $excerpt;
         }
 
-        $more = '<!-- more -->';
+        $more = Config::get('rainlab.blog::summary_separator', '<!-- more -->');
         if (strpos($this->content_html, $more) !== false) {
             $parts = explode($more, $this->content_html);
             return array_get($parts, 0);
         }
 
-        return Html::limit($this->content_html, 600);
+        $length = Config::get('rainlab.blog::summary_default_length', 600);
+        return Str::limit(Html::strip($this->content_html), $length);
```

Comment:
This may break existing installations as this is now removing support for summaries to have HTML tags within them.

Reply:
@tobias-kuendig

Reply:
Good catch. I have changed it back to `Html::limit`.